vendors/cypress/boards/CY8CKIT_064S0S2_4343W/aws_demos/config_files/iot_config.h

```diff
@@ -60,6 +60,12 @@
 /* Platform thread priority. */
 #define IOT_THREAD_DEFAULT_PRIORITY             ( 5 )
 
+/* Platform stack size for HTTPS library. */
```

Comment:
The comment only mentions about stack size, but the configurations seems to be for task count, queue size and static memory.